src/parseTools.js

```diff
@@ -96,7 +96,7 @@ function preprocess(text, filenameHint) {
               showStack.push(truthy ? SHOW : IGNORE);
             } else if (first === '#include') {
               if (showCurrentLine()) {
-                let filename = line.substr(line.indexOf(' ') + 1);
```

Comment:
unrelated bugfix?

Reply:
This is a related bugfix: I've indented the `#include` statement but that won't work without this:
```
     #include "polyfill/bigint64array.js"
```